### DISCUSSION TITLE
Is it possible to select/copy the contents of a ScrollView widget?

### DISCUSSION THREAD

**Discussion by jeffwright13** [Textualize/textual#299] [Q&A] [answered]
I have an application that renders rich Text in a ScrollView object. This works great for displaying text. But is there a way to select/copy the text (using the mouse and Ctrl-C)?

**Comment by jeffwright13** [comment, marked as answer]
Anyone know the answer?

**Reply by willmcgugan** [reply]
Terminals aren't great for this. We have a solution in the works, but for now you can try holding alt and drag to select.

**Reply by jeffwright13** [reply]
No, that's a perfectly fine answer. It also works on Mac using the Option key.

**Reply by James4Ever0** [reply]
i suggest you to 'toggle off' mouse capturing completely on receiving keycode like 'c', not just stop sending message to widgets but disable this behavior

**Reply by James4Ever0** [reply]
holding 'alt' or 'option' key does work in conventional terminals, but does not work in vscode builtin terminal.
